models/src/main/java/com/vimeo/networking2/Video.kt

```diff
@@ -120,6 +120,12 @@ data class Video(
     @Json(name = "name")
     val name: String? = null,
 
+    /**
+     * Information about the folder that contains the video, or null if it is in the root folder.
```

Comment:
How does "directory" sound instead of "location?" I used "root folder" in both of these docs because it's a common phrase used in file systems. I can understand that "root folder" is a little confusing as the root folder isn't actually a `Folder` in our case, but "root location" isn't really a familiar phrase used in file systems in my experience. I think "root directory" strikes a balance between familiarity and avoiding the confusion of "root folder" not being an actual `Folder`.

Reply:
Yeah, I like that. 👍